tuf/repository_lib.py

```diff
@@ -608,8 +553,9 @@ def _load_top_level_metadata(repository, top_level_filenames, repository_name):
     tuf.roledb.update_roleinfo('timestamp', roleinfo, mark_role_as_dirty=False,
         repository_name=repository_name)
 
-  else:
-    logger.debug('Cannot load the Timestamp  file: ' + repr(timestamp_filename))
+  except securesystemslib.exceptions.StorageError:
+    raise tuf.exceptions.RepositoryError('Cannot load the Timestamp  file: '
```

Comment:
Tiny nit (looks like it was carried over from the old string): double spacing between `Timestamp` and `file`.

Reply:
Good catch, thanks. Corrected in a187377.